### ISSUE TITLE
[BUG] NullPointerException: Profile name must not be null on updating to 1.20.2

### ISSUE THREAD

**Issue by DrPepplies** [snowgears/shopbugs#417]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Type of bug

Compatibility issue, Error in console

### Environment

```markdown
- Server Version: Paper v1.20.2 (build 234, latest)
- Shop Version: v1.8.4.5
```

### Server startup log

It's almost entirely the error

### Shop config files

https://paste.gg/p/anonymous/c98bcca929c743f09207e8f8bb76656c

### Error log (if applicable)

https://paste.gg/p/anonymous/10af753e0a324fa784aac7b312db500b

### Bug description

Shop works perfectly on 1.20.1 with Shop 1.8.4.4, but on upgrading to 1.20.2 and the new version of Shop, this error spams the console thousands of times.

Haven't changed any config or setup between the updates. No other updated plugins are breaking so I don't think it's a general 1.20.2 problem.

I've included two versions of the error, it logs the first one occasionally but the second one *tons*. I assume it's the same thing but is happening in a repeating task.

All the shops say null on the owner line, and the chests can be opened by anybody.

### Steps to reproduce

Don't know, started happening on Minecraft update.

### Expected behaviour

No error

### Actual behaviour

Error

**Comment by DrPepplies** [comment]
I've experimented a bit more to try and find out why this happens, I think it's unable to resolve UUIDs for people's usernames anymore? When I create my own shops on the server, they work fine. When I import only my own data from the 1.20.1 server it seems to work fine (even before I log in). But then if I try pulling other people's data files across, even one single file with only one shop in it, the error happens and the shop says "null" on the owner sign.

I don't think I've changed anything related to that.
